README.md

```diff
@@ -6,5 +6,14 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
-1. Using a marker, pen, or pencil, sketch the components of an (R)MVC architecture, how they interconnect, and which components have what responsibilities. Take a photo of your drawing and add it to the `submissions` directory inside your clone.
-1. Describe the flow of a PATCH request as it goes through the app.
+  Rails exists to help create a back-end framework for web development. It makes writing web applications easier compared to creating everything on your own.
+
+
+2. Using a marker, pen, or pencil, sketch the components of an (R)MVC architecture, how they interconnect, and which components have what responsibilities. Take a photo of your drawing and add it to the `submissions` directory inside your clone.
+
+  See submission folder (very basic version of relations)
+
```

Comment:
Right in some respects, but off on a couple of major points.
1. Where is the Server? The user doesn't interact directly with a controller.
2. The view is not (insofar as we've discussed it) a standalone component in an MVC app. Rather, the controller uses data that it receives from the model to build the 'view', and then sends it back to the server.
3. Where is the DB?